index.html

```diff
@@ -2855,6 +2858,87 @@ <h2 class="icon-title">
           </div>
         </section>
       </section>
+      <section>
+        <h2>
+          Single color icons
+        </h2>
+        <p>
+          Some platforms enforce that icons be displayed with a <dfn>single
+          color</dfn>, where only the transparency of the icon can be
+          controlled. As web applications need to across multiple
+          platforms, it is possible to indicate that an icon can have a
+          user-agent-specified color applied by adding the <a>single-color</a>
```

Comment:
User Agents can use different colors depending on where they display it - they could even use a gradient - it is really about masking that allows them to do so

Reply:
For this reason the name "single-color" might be wrong.

Reply:
Do you have any preferences on the names previously suggested? (`monochrome`, `greyscale`, `stamp`)

Reply:
The **purpose** of this icon is that it will be use as **monochrome** - sounds wrong.

Maybe silhouette?

Reply:
Apart from this I like monochrome more than single-color

Reply:
Maybe "mask" or "image-mask" might be the most descriptive

Reply:
cut-out

1. a shape cut out of board or another material.
    "a life-size cut-out"
2. a hole cut in something for decoration or to allow the insertion of something else.
     "the mains output socket is fitted into a suitable cut-out on the rear panel"

Reply:
I think that mask, image-mask, and cut-out could be confused with maskable. Image masking is like cutting out parts of the image that aren't inside the mask. I'd like to steer away from mask/cut terminology.

Similarily, I think you could argue "the **purpose** of this icon is that it will be used as **maskable**" sounds just as odd. I think you could rephrase it as:
- The purpose of this icon is that it is maskable
- The purpose of this icon is that it is monochrome

I like silhouette but it doesn't fit that dichotomy. I'm going to switch back to monochrome as that has the most consensus.